presto-docs/src/main/sphinx/functions/teradata.rst

```diff
@@ -11,7 +11,7 @@ String Functions
 
     Returns the hexadecimal representation of the UTF-16BE encoding of the string.
 
-.. function:: index(string, substring) -> bigint
+.. function:: index(string, substring, instance) -> bigint
```

Comment:
see https://www.info.teradata.com/HTMLPubs/DB_TTU_16_00/SQL_Reference/B035-1145-160K/gcn1472241145389.html#wwID0ENTAM

Reply:
@findepi thanks for the doc. In that case, I will have to decouple Teradata's index() from strpos(). Is there a way to make an argument of strpos() optional? Or, should I implement a new method altogether?

Reply:
I have undone the changes related to Terdata index() function and added a separate stringPosition() method that takes instance as an argument. Please review the changes.